api/src/pcapi/routes/serialization/venues_serialize.py

```diff
@@ -507,6 +508,18 @@ class AdageCulturalPartnerResponseModel(BaseModel):
     id: int
     statutId: int | None
     siteWeb: str | None
+    domaineIds: list[int]
```

Comment:
@validator("domaineIds", pre=True)
def transform_domaine_ids(cls, domaine_ids: str | list[int] | None) -> list[int]:
  if not cultural_partner.domaineIds:
    return []
  
  if isinstance(domaine_ids, list):
    return domaine_ids
    
  return [id for id in cultural_partner.domaineIds.split(",")]